test/unit/test.speech_to_text.v1.js

```diff
@@ -223,6 +223,27 @@ describe('speech_to_text', function() {
     });
   });
 
+  describe('recognizeWebM()', function() {
+    const session_path = '/v1/sessions/foo/recognize';
+    const payload = {
+      audio: fs.createReadStream(__dirname + '/../resources/sample1.webm'),
```

Comment:
it may be worth creating a new variable here to hold the contents of this file instead of re-reading it for every test, though.